SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.csproj

```diff
@@ -43,7 +43,7 @@
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
     <PackageReference Include="Improbable.SpatialOS.Platform">
-      <Version>13.7.1</Version>
+      <Version>14.2.1</Version>
```

Comment:
SpatialOS `14.0.0` introduced [breaking changes](https://docs.improbable.io/reference/14.2/releases/release-notes#breaking-changes).

If any of these breaking changes are applicable to GDK customers please explicitly call them out in the CHANGELOG and in release comms.

Reply:
I'll roll this back for 0.6